_includes/js/main.js

```diff
@@ -1,7 +1,9 @@
 import { Slider } from "./slider.js";
 import { Locations } from "./locations";
+import { SlideIn } from "./animate-slidein";
 
-(() => {
+window.onload = () => {
```

Comment:
Please don't use an Event Listener here. This caused loading problems and was fixed this morning on the development branch. If you need to listen to the load event of the window object, you'll need to do that in your component.
```suggestion
(() => {
```